Sources/REventLogger/REventLoggerEnvironment.swift

```diff
@@ -25,7 +26,11 @@ final class REventLoggerEnvironment {
     }
 
     var appName: String {
-        bundle.value(for: BundleKeys.displayName) ?? bundle.valueNotFound
+        if let bundleDisplayName = bundle.value(for: BundleKeys.bundleDisplayName) {
```

Comment:
ignore the if condition and add one more value with nil coalescing `bundle.value(for: BundleKeys.bundleDisplayName) ?? bundle.value(for: BundleKeys.displayName) ?? bundle.valueNotFound`